packages/plugins/ansible-runner/ansible-runner.spec

```diff
@@ -57,15 +58,19 @@ ln -s %{_bindir}/ansible-runner-%{python3_version} %{buildroot}/%{_bindir}/ansib
 
 %files
 %defattr(-,root,root)
-
-%files -n python%{python3_pkgversion}-%{pypi_name}
-%{python3_sitelib}/*
 %{_bindir}/ansible-runner
 %{_bindir}/ansible-runner-3
 %{_bindir}/ansible-runner-%{python3_version}
+
+%files -n python%{python3_pkgversion}-%{pypi_name}
+%{python3_sitelib}/*
 %{_datadir}/ansible-runner/*
```

Comment:
Shouldn't the `datadir` be also be in the "normal" (non-`python%{python3_pkgversion}`) package then, as it is also not-python-specific?

Reply:
I wondered about that, I wasn't sure what exactly the datadir was used for or what is put in there.